docs/en/development/adding_test_queries.md

```diff
@@ -126,6 +126,17 @@ clickhouse-client -nmT < tests/queries/0_stateless/01521_dummy_test.sql | tee te
 - use other SQL files in the `0_stateless` folder as an example
 - ensure the feature / feature combination you want to test is not yet covered with existing tests
 
+#### Test naming rules
+
+It's important to name tests correctly, so one could turn some tests subset off in clickhouse-test invocation.
+
+| Tester flag| What should be in test name | When flag should be added | Example of test with flag |
+|---|---|---|---|
+| `--[no-]zookeeper`| "zookeeper" or "replica" | Test uses tables from ReplicatedMergeTree family | |
+| `--[no-]shard` | "shard" or "distributed" or "global"| | |
+| `--[no-]long` | "long" or "deadlock" or "race" | Test runs longer than TODO seconds | |
+| `--[no-]pbi `| "pbi" | Test exploits Possibly Broken Invariants e.g. that CH is located in /var/lib/clickhouse | |
```

Comment:
Ok, but I don't like `pbi` abbr.
Let's name it `invariant`.

Reply:
> that CH is located in /var/lib/clickhouse

What does it mean? ClickHouse data is not necessarily located in `/var/lib/clickhouse`.

Reply:
Yup, but some tests assume it is